R/RLearner_classif_liquidSVM.R

```diff
@@ -0,0 +1,49 @@
+#' @export
+makeRLearner.classif.liquidSVM = function() {
+  makeRLearnerClassif(
+    cl = "classif.liquidSVM",
+    package = "liquidSVM",
+    par.set = makeParamSet(
+      makeIntegerLearnerParam(id = "d", lower = 0L),
+      makeLogicalLearnerParam(id = "scale"),
+      makeIntegerLearnerParam(id = "threads", lower = -1L),
+      makeIntegerLearnerParam(id = "partition_choice", lower = 0L),
+      makeIntegerLearnerParam(id = "grid_choice", lower = -2L, upper = 2L),
```

Comment:
How does this interact with `gamma_steps`, `lambda_steps`, etc? Are these options mutually exclusive?

Reply:
And all of these options should be not tunable.